### DISCUSSION TITLE
student developer tool not working & apply properly

### DISCUSSION THREAD

**Discussion by Gautam5514** [community/community#136837] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

In my Github there is a problem with using the student developer tool, Actually It Will not working properly and not take the benefit of my student Id to get the student developer tool 
Please check this and make this usable
Moderators Edit: I've removed your private information from [this post](link to discussion) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by davevad93** [comment, marked as answer]
Hi @Gautam5514 ,

The problem may be that your ID doesn't show a current date, it does say `Valid until 30.08.2025` but from my previous experience the application system requires a current date as proof otherwise it rejects you. 

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

So try one of the above documents with a current date proving your current enrollment in `DD/MM/YYYY` or `MM/DD/YYYY` format.

**Reply by Gautam5514** [reply]
Thank you
